zerver/webhooks/appfollow/view.py

```diff
@@ -14,13 +14,13 @@
 @api_key_only_webhook_view("AppFollow")
 @has_request_variables
 def api_appfollow_webhook(request, user_profile, stream=REQ(default="appfollow"),
-                          payload=REQ(argument_type="body")):
-    # type: (HttpRequest, UserProfile, Text, Dict[str, Any]) -> HttpResponse
+                          topic=REQ(default=None), payload=REQ(argument_type="body")):
+    # type: (HttpRequest, UserProfile, Text, Optional[Text], Dict[str, Any]) -> HttpResponse
     message = payload["text"]
     app_name = re.search('\A(.+)', message).group(0)
 
-    check_send_stream_message(user_profile, request.client, stream,
-                              app_name, convert_markdown(message))
+    check_send_stream_message(sender=user_profile, client=request.client, stream_name=stream,
+                              topic=(topic or app_name), body=convert_markdown(message))
```

Comment:
I'd do this on its own line with `if topic is None:` (in general, we try to avoid use of falsyness in production code, because it combines together the totally different concepts of None and ''.  Like, what if the user wanted to use a topic of ''?  You'd need to think a while to figure out whether this code is broken in that case) and also add a test for it to maintain full test coverage.

Reply:
Okay, makes sense.